docs/platform/howto/manage-domains.rst

```diff
@@ -0,0 +1,87 @@
+Manage domains
+===============
+
+Verified domains let you manage users in your organization and set up security policies.
+
+There are two ways you can verify a domain:
+
+* by adding a DNS TXT record to the domain (recommended)
+* by uploading an HTML file to the domain's root folder
+
+.. note::
+    After adding a domain, your organization users will automatically become managed users. Managed user features will be available as part of a future release.
+
+Add a domain using a DNS TXT record
+-------------------------------------
+
+#. In the organization you want to add a domain to, click **Admin**.
+
+#. Click **Domains**.
+
+#. Click **Add domain**.
+
+#. Enter a **Domain name**.
+
+#. In the **Verification method**, select **Add a DNS TXT record to your domain host**.
+
+#. In another browser tab or window, log in to your domain hosting provider.
+
+#. In the domain hosting provider, go to the DNS settings.
+
+#. In the Aiven Console, copy the TXT record value.
+
+#. In the DNS settings, create the TXT record with the following:
+
+.. list-table::
+      :header-rows: 1
+      :align: left
+
+      * - Field name
+        - Value
+      * - Name
+        - ``_aiven-challenge.{your domain}``
+      * - Value 
+        - The TXT record value shown in Aiven Console.
+      * - Type
+        - ``TXT``
+
+#. In the Aiven Console, click **Add domain**.
```

Comment:
After the domain is setup the customer will have to click the "three dots" in the row of the domain they want to verify and there will be a button with text "Verify", that's the one to be pressed to trigger the verification itself.